packages/react-router/lib/router/router.ts

```diff
@@ -2722,13 +2725,13 @@ export function createRouter(init: RouterInit): Router {
       results = await callDataStrategyImpl(
         dataStrategyImpl,
         type,
-        state,
         request,
         matchesToLoad,
         matches,
         fetcherKey,
         manifest,
-        mapRouteProperties
+        mapRouteProperties,
+        routerContext
```

Comment:
This is the core functional addition - the shared `dataStrategy` stuff already supports `context` for SSR, we just didn't used to pass anything for client side usages - now we do